### PR TITLE
update set mode gpu on the new interface

### PR DESCRIPTION
**Pull request by Lupino**
Fixed. caffe.Classifier unexpected keyword argument 'gpu'.

### CODE REVIEW

File: python/classify.py [hunk 1]
```diff
@@ -98,11 +98,12 @@ def main(argv):
 
     # Make classifier.
     classifier = caffe.Classifier(args.model_def, args.pretrained_model,
-            image_dims=image_dims, gpu=args.gpu, mean=mean,
+            image_dims=image_dims, mean=mean,
             input_scale=args.input_scale, raw_scale=args.raw_scale,
             channel_swap=channel_swap)
 
     if args.gpu:
+        caffe.set_mode_gpu()
```
Comment (longjon) [inline python/classify.py]: Shouldn't this be done before constructing the `Net` above?

**Comment by Lupino** [comment]
https://github.com/BVLC/caffe/issues/1937

**Comment by tkng** [comment]
:+1, I encountered the same problem.

**Comment by shelhamer** [comment]
Thanks for noting the issue and sending a fix -- I'm collecting these fixes and other pycaffe example changes in #2058.

**Pull request closed without merging**